specification/synapse/resource-manager/Microsoft.Synapse/stable/2020-12-01/examples/Libraries_ListByWorkspace.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-12345678abc",
+    "resourceGroupName": "exampleResourceGroup",
+    "workspaceName": "exampleWorkspace",
+    "ifNoneMatch": null,
+    "api-version": "2019-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Date": "Wed, 23 Oct 2020 21:33:05 GMT",
+        "X-Content-Type-Options": "nosniff",
+        "x-ms-ratelimit-remaining-subscription-reads": "14909",
+        "x-ms-request-id": "f632df92-1140-4e19-9e19-0b1c8cbe393b",
+        "x-ms-correlation-request-id": "f632df92-1140-4e19-9e19-0b1c8cbe393b"
+      },
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/exampleResourceGroup/providers/Microsoft.Synapse/workspaces/exampleWorkspaceName/libraryartifacts/exampleLibraryName1.jar",
```

Comment:
The API path was "/libraries/", not "/libraryartifacts/"

Reply:
The list API is a proxy API that returns the response from a different service called artifact service, which adds 'artifacts' at the end of the resource type.

Reply:
That is not appropriate for an ARM API and will fail validation resulting in S360 alerts for your service.  See https://armwiki.azurewebsites.net/api_contracts/RPCstrongvalidation.html?q=rpc%20vali

Reply:
I'll work with artifact service team and try to fix this issue.